### DISCUSSION TITLE
PnP modern search result webpart in SharePoint Online: how to get group items by property run?

### DISCUSSION THREAD

**Discussion by stefanbaader** [microsoft-search/pnp-modern-search#3469] [Q&A]
Hello,
I am struggling with a specific requirement: I want to create a result for just some link content types, grouped by the column "LinkCategoryOWSTEXT". 
The results should be DIV elements with a flex style (so responsive) and the elements should be grouped by a category DIV element.

"slotName": "LinkCategoryOWSTEXT",
"slotField": "LinkCategoryOWSTEXT"

The example in the documentation is
{{#group items by="property"}}

but I can't get it run.
I am using that simple example in https://microsoft-search.github.io/pnp-modern-search/scenarios/create-your-first-custom-template/#change-the-content-of-your-new-template-file and this works fine.

But I want to change that into div elements which are grouped by a category.

Any idea?

I add my example JSON template code which works fine without "group items by". I commented that out.

[example-template.JSON](https://github.com/microsoft-search/pnp-modern-search/files/14021392/example-template.JSON)

**Comment by wobba** [comment]
Have you added LinkCategoryOWSTEXT to the list of properties to retrieve in the web part config? And you see values coming back for the property?
